src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,14 @@
 package mate.academy.service;
 
-public class AuthenticationService {
-    /**
-     * Imagine that some user wants to login to your site.
-     * You should check if user credentials (login and password) are valid or not.
-     * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
-     * @param password - user's password
-     * @return true if user by email exists and passed password is equal to user's password.
-     * Return false in any other cases.
-     */
+import mate.academy.model.User;
+
+public class AuthenticationService extends UserService {
```

Comment:
let's not extend UserService

we can use `findByEmail()` if we create `new UserService()` object inside `login` method